base/applications/mmc/precomp.h

```diff
@@ -32,6 +32,11 @@ INT
 LengthOfStrResource(IN HINSTANCE hInst,
```

Comment:
While here, this function does not need to be exported. And make it `static` in misc.c.

Reply:
:/

![image](https://user-images.githubusercontent.com/610685/187082214-6dd5f777-9a39-4bcf-97e8-f0a0b7fac5d8.png)

please read it again.
https://github.com/reactos/reactos/blob/master/PULL_REQUEST_MANAGEMENT.md